docs/core/whats-new/dotnet-9/overview.md

```diff
@@ -53,9 +53,58 @@
 
 For more information, see [What's new in the SDK for .NET 9](sdk.md).
 
+## AI Building Blocks and Fundamentals
+
+### Microsoft.Extensions.AI & Microsoft.Extensions.VectorData
+
+.NET 9 introduces a unified layer of C# abstractions through [Microsoft.Extensions.AI](https://www.nuget.org/packages/Microsoft.Extensions.AI.Abstractions/) and [Microsoft.Extensions.VectorData](https://www.nuget.org/packages/Microsoft.Extensions.VectorData.Abstractions/). These abstractions facilitate interaction with AI services, including small and large language models (SLMs and LLMs), embeddings, vector stores, and middleware.
+
+### Tokenizers
+
+The [Microsoft.ML.Tokenizers](https://www.nuget.org/packages/Microsoft.ML.Tokenizers) library provides .NET developer with capabilities for encoding and decoding text to tokens. For AI scenarios, this is important to manage context, calculate cost, and pre-process text when working with local models.
+
+The latest release introduces significant new capabilities:
+
+- Tokenizers
+  - Byte-Level BPE
+  - SentencePiece
+  - WordPiece
```

Comment:
Please list these as:

- Tiktoken
- Llama (based on SentencePiece)
- CodeGen
- Phi2 (based on CodeGen)
- WordPiece 
- Bert (based on WordPiece)

Reply:
@tarekgh do you want to suggest the edits for this? I will accept them in what you have here looks good just want to make sure you get it looking lik eyou want

Reply:
I added the suggestions. Sorry for the multiple as somehow the line selections didn't work when I added the first.